modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/controls/OrderDropdown.tsx

```diff
@@ -0,0 +1,133 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import Button from '@clayui/button';
+import DropDown from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+import React, {useContext, useEffect, useState} from 'react';
+
+import {TSorting} from '../../index';
+import ViewsContext from '../../views/ViewsContext';
+
+// @ts-ignore
+
+import {VIEWS_ACTION_TYPES} from '../../views/viewsReducer';
+
+function OrderDropdown() {
```

Comment:
Yup, I can rename it to `SortDropdown` since all these pieces are tied closely. I usually like naming components closer to how they appear in the UI since when I go to maintain code I look at the UI and it makes it easier to find which component it's related to.